xbmc/guilib/GUITexture.cpp

```diff
@@ -27,6 +30,35 @@ CTextureInfo::CTextureInfo(const std::string &file):
   useLarge = false;
 }
 
+void CGUITexture::Register(const CreateGUITextureFunc& createFunction,
+                           const DrawQuadFunc& drawQuadFunction)
+{
+  m_createGUITextureFunc = createFunction;
+  m_drawQuadFunc = drawQuadFunction;
+}
+
+CGUITexture* CGUITexture::CreateTexture(
+    float posX, float posY, float width, float height, const CTextureInfo& texture)
+{
+  if (!m_createGUITextureFunc)
+    throw std::runtime_error(
```

Comment:
I think `<cstdexcept>` should be included

Reply:
Added 🙂